jwst/associations/asn_gather.py

```diff
@@ -35,6 +36,10 @@ def asn_gather(association, destination=None, exp_types=None, exclude_types=None
     exclude_types : [str[,...]] or None
         List of exposure types to exclude.
 
+    source_folder : str or None
+       Folder where the members originate from.
```

Comment:
I guess we can let you get away with ending a sentence with a preposition, after all it's only a doc string ...  :-)

Reply:
🤦